gems/smithy/lib/smithy/anvil/client/views/endpoint_parameters.rb

```diff
@@ -9,8 +9,8 @@ class EndpointParameters < View
           def initialize(plan)
             @plan = plan
             @model = plan.model
-            service = @plan.service
-            @endpoint_rules = service.values.first['traits']['smithy.rules#endpointRuleSet']
+            _, service = @plan.service.first
```

Comment:
I still find this confusing, but if we're keeping the shape as a hash of id => data, I prefer the orig code to using a `_, service =` personally.